bands/templates/bands/bio.html

```diff
@@ -63,9 +63,9 @@
     {{ macros.form_group(
         band.bio,
         'teaser_song_url',
-        label='Teaser Song',
+        label='Media Link',
         placeholder='www.example.com/teaser-song.mp3',
-        help='Please provide a cloud storage link to an .mp3 of the song you would featured in a "teaser" to be shared via MAGFest social media') }}
+        help='Please provide a cloud storage link to either a website you would like to link to or an .mp3 of the song you would like featured in a "teaser" to be shared via MAGFest social media') }}
```

Comment:
For what it's worth, a lot of the descriptions were explicitly requested as-is by the music department. See https://github.com/magfest/bands/issues/85 for details. The ultimate intent of this field may be so the music department can download an .mp3, thus a link to a streaming site might not actually work.

Although, there was definitely a typo in the original: "would **like** featured"